File: faculdade/semestre-01/programacao-de-computadores/exercicios/aula-02-atribuicao-entrada-e-saida/exerc04.py
#Escreva um programa em python que calcule as duas raizes de uma equação de 2º Grau ax**2+bx+c. conhecendo os valores dos coeficientes da mesma. Suponha que as raizes são reais.


a = float(input("Digite o coeficiente a: "))
b = float(input("Digite o coeficiente b: "))
c = float(input("Digite o coeficiente c: "))

delta = b**2 - 4*a*c

raiz_delta = delta**0.5

x1 = (-b + raiz_delta) / (2*a)
x2 = (-b - raiz_delta) / (2*a)

print ("O valor de de x1 é: ", x1, "e o valor de x2 é: ", x2)

File: Java/exercicios-diversos/exercicios-diversos/src/exercicios/exerc04.java
//Escreva um programa que calcule as duas raizes de uma equação de 2º Grau ax**2+bx+c. conhecendo os valores dos coeficientes da mesma. Suponha que as raizes são reais.

package exercicios;
import javax.imageio.ImageTranscoder;
import java.util.Scanner;
public class exerc04 {
    static void main() {
        Scanner sc = new Scanner(System.in);
        System.out.print("Digite o valor de a: ");
        int valorA = sc.nextInt();
        System.out.print("Digite o valor de b: ");
        int valorB = sc.nextInt();
        System.out.print("Digite o valor de c: ");
        int valorC = sc.nextInt();

        double delta = Math.pow(valorB, 2) - 4*valorA*valorC;

        double raizDelta = Math.sqrt(delta);

        double x1 = (-valorB + raizDelta)/(2*valorA);
        double x2 = (-valorB - raizDelta)/(2*valorA);

        System.out.println("O valor do x1 é: "+x1+" e o valor do x2 é: "+x2);
    }
}
